sw/device/lib/dif/dif_aes_unittest.cc

```diff
@@ -60,9 +60,10 @@ class AesTestInitialized : public AesTest {
   dif_aes_t aes_;
 
   const dif_aes_transaction_t kTransaction = {
+      .operation = kDifAesOperationEncrypt,
+      .mode = kDifAesModeEcb,
       .key_len = kDifAesKey128,
-      .mode = kDifAesModeEncrypt,
-      .operation = kDifAesOperationAuto,
+      .manual_operation = kDifAesManualOperationAuto,
```

Comment:
Maybe it's better to have this be a `dif_toggle_t` type, rather than a separate enum? Then this field could just be whether manual operation is "enabled" or "disabled". As is, it a tad confusing to have two enum types `*_operation_t` and `*_manual_operation_`.

Reply:
This sounds indeed like an option. But I am wondering if it isn't simpler to keep it following the hardware. Anyway, I've added your proposal to corresponding issue #10487. I will leave it as is for now to move forward and unblock DV folks waiting on this PR.